pkg/component/plutono/plutono.go

```diff
@@ -135,13 +150,16 @@ func (p *plutono) Deploy(ctx context.Context) error {
 	}
 
 	// dashboards configmap is not deployed as part of MR because it can breach the secret size limit.
-	_, err = controllerutils.GetAndCreateOrMergePatch(ctx, p.client, dashboardConfigMap, func() error {
-		metav1.SetMetaDataLabel(&dashboardConfigMap.ObjectMeta, "component", name)
-		metav1.SetMetaDataLabel(&dashboardConfigMap.ObjectMeta, references.LabelKeyGarbageCollectable, references.LabelValueGarbageCollectable)
-		return nil
-	})
-	if err != nil {
-		return err
+	for _, configMap := range dashboardConfigMap {
```

Comment:
```suggestion
	for _, configMap := range dashboardConfigMaps {
```

I suggest renaming the variable to reflect that it is now a slice of configmap pointers.